sklearn/neural_network/tests/test_mlp.py

```diff
@@ -175,9 +175,10 @@ def test_gradient():
     # are correct. The numerical and analytical computation of the gradient
     # should be close.
     for n_labels in [2, 3]:
-        n_samples = 5
+        n_samples = 100
```

Comment:
I don't see what this helps. If it worked with 5 and that tests the functionality we seek, why change it?

Reply:
Sorry, My bad, I suggested it to make the test more robust now that the random seed is fixed, but I was misled in thinking that the gradient was taken w.r.t the samples (too much gradient boosting I guess.)

`numgrad` and `grad` size is already 121 or 143 so this is not needed. Can you please revert @aditya1702  ?

LGTM otherwise

Reply:
@NicolasHug Reverted